index.html

```diff
@@ -114,21 +114,21 @@ <h2 class="section-heading">Pump up skills</h2>
                     <div class="item-wrap">
                         <div class="icon pull-xs-left"><i class="zmdi zmdi-file-text zmdi-hc-3x"></i></div>
                         <div class="item">
-                            <h3><a href="#" target="_blank">GitHub</a></h3>
+                            <h3><a href="https://services.github.com/kit/downloads/github-git-cheat-sheet.pdf" target="_blank">GitHub</a></h3>
```

Comment:
Sorry I sent the wrong link! https://services.github.com/kit/downloads/github-git-cheat-sheet.pdf